roadmap-reviews/2020/end_of_august_19.md

```diff
@@ -36,8 +36,8 @@ reference implementation to our golang implementation.
 ### Jenkins
 
 Our jenkins implementation has mostly seen work on cosmetic changes, as well as
-simplifying the build procedures. We expect the rest of the quarters to take on
-the standard stream capturing features.
+simplifying the build procedures. We expect the rest of the roadmap period to
+take on the standard stream capturing features.
```

Comment:
I think this needs a minor fix:
`We expect to take on the standard stream capturing features later this year.`
or something of that sort.

Other than this, it LGTM :)